src/main/java/com/littleBeasts/entities/Player.java

```diff
@@ -133,4 +144,79 @@ public void setFighting(boolean fighting) {
     public boolean isFighting() {
         return isFighting;
     }
+
+    public GameLogic getGameLogic() {
+        return gameLogic;
+    }
+
+    public void setGameLogic(GameLogic gameLogic) {
+        this.gameLogic = gameLogic;
+    }
+
+    @Action(description = "This is a punch, it hurts.")
+    public void punch() {
```

Comment:
For now ok. But if we add attacks we should change this. This should be Part of another Class e.g. PlayerAnimations, which will be more easy to maintain and will also improve readability.

Reply:
Yeah, it will be moved to the Attack

Reply:
I already talked with Lawand about that. Perhaps we can have different Kategories of Attacks, like Melee, Ranged and Spell, with their own animations.